include/deal.II/base/config.h.in

```diff
@@ -39,6 +39,9 @@
 #cmakedefine DEAL_II_WITH_ARPACK
 #cmakedefine DEAL_II_WITH_ARBORX
 #cmakedefine DEAL_II_WITH_ASSIMP
+#ifdef FEATURE_BOOST_BUNDLED_CONFIGURED
+#  define DEAL_II_WITH_BOOST_BUNDLED
+#endif
```

Comment:
That really just hides the problem, you're still using the non-namespaced name. You'll have to do the translation within cmake already, and then have something like this here:
```suggestion
#cmakedefine DEAL_II_WITH_BOOST_BUNDLED
```

Reply:
You're right. 🙁 I decided to simply prefix all `FEATURE_*_BUNDLED_CONFIGURED` variables with `DEAL_II` and use `DEAL_II_FEATURE_BOOST_BUNDLED_CONFIGURED` then.